app/views/content/teaching-internship-providers.md

```diff
@@ -48,10 +52,23 @@ providers:
       link: "https://www.setsa.info/1569/news-1/post/20/teaching-internship-2021"
       name: Emma Lowne
       email: e.lowne@setsa.info
+    - header: The Becket School 
```

Comment:
have you put Becket in East of England instead of East Midlands for a reason? Is the excel spreadsheet wrong?

Reply:
Nope, that's an error. ~ood catch.